### PR TITLE
README: simplify setup, use PostgreSQL local auth by default

### PR DESCRIPTION
**Pull request by seirl**
With PostgreSQL, it is unnecessary to define database passwords for a
local development environment. We can simply use the "local" auth of
PostgreSQL, which allows UNIX users to connect without password to the
PG users of the same name. This greatly improves the ease of setup, and
removes the need for a config.sh.

### CODE REVIEW

**Comment by seirl** [comment]
This obsoletes #651 .

File: mangaki/mangaki/settings.py [hunk 1]
```diff
@@ -109,10 +109,10 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': config.get('pgsql', 'DB_NAME', fallback='mangaki'),
-        'USER': config.get('pgsql', 'DB_USER', fallback='django'),
-        'PASSWORD': config.get('secrets', 'DB_PASSWORD'),
-        'HOST': config.get('pgsql', 'DB_HOST', fallback='127.0.0.1'),
-        'PORT': '5432',
+        'USER': config.get('pgsql', 'DB_USER', fallback=''),
```
Comment (RaitoBezarius) [inline mangaki/mangaki/settings.py]: Is there any rationale behind this? `django` looks like a sane default to me

File: mangaki/mangaki/settings.py [hunk 2]
```diff
@@ -109,10 +109,10 @@
...
+        'PASSWORD': config.get('secrets', 'DB_PASSWORD', fallback=''),
+        'HOST': config.get('pgsql', 'DB_HOST', fallback=''),
+        'PORT': config.get('secrets', 'DB_PORT', fallback=''),
```
Comment (RaitoBezarius) [inline mangaki/mangaki/settings.py]: Is `DB_PORT` really a secret?

File: mangaki/mangaki/settings.py (hunk 1 above)
Comment (seirl) [reply mangaki/mangaki/settings.py]: Yes, allowing people to use PostgreSQL local auth, as specified in the commit message. USER, HOST and PASSWORD have to be empty for it to work.

File: mangaki/mangaki/settings.py (hunk 2 above)
Comment (seirl) [reply mangaki/mangaki/settings.py]: Good catch!

Comment (seirl) [reply mangaki/mangaki/settings.py]: Fixed.

**Comment by jilljenn** [comment]
OK mais j'ai juste peur que sur le serveur, il y ait un problème lors de la prochaine mise à jour car il n'y aura plus eu besoin de mot de passe. OK ?

**Pull request merged**